template/_gitignore

```diff
@@ -44,6 +44,7 @@ yarn-error.log
 buck-out/
```

Comment:
There's a removal of Vscode section from gitignore as well (at least when I tried to copy the template)

Reply:
Haha, yes, the VS Code section is an addition I did to the template as most of the TS developers use VS code. I guess if the goal is to stay consistent with the default template as much as possible, I should remove it, hmm?

Reply:
Ok then it's fine if you did it